src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -218,6 +235,20 @@ private String startDockerContainerWithTerraInit(
     return startDockerContainer(fullCommand, workingDir, envVars, bindMounts);
   }
 
+  // ====================================================
+  // Docker containers
+
+  /** Build the Docker client object with standard options. */
+  private void buildDockerClient() {
+    DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder().build();
+    DockerHttpClient httpClient =
+        new ApacheDockerHttpClient.Builder()
+            .dockerHost(config.getDockerHost())
+            .sslConfig(config.getSSLConfig())
+            .build();
+    dockerClient = DockerClientImpl.getInstance(config, httpClient);
```

Comment:
WDYT about returning the dockerClient from this function? If it's initialized here and not valid to use before calling this function, returning the dockerClient and using it as a local variable instead of a class member makes ordering  dependency clear. Or move it to the DockerAppsRunner constructor?

Reply:
Good idea. Moved it to the constructor.